.travis.yml

```diff
@@ -5,7 +5,9 @@ python:
   - 3.8
   - 3.7
   - 3.6
-  - 3.5
+
+before_install:
+  - sudo apt-get -y install python3-openslide
```

Comment:
This https://github.com/TIA-Lab/tiatoolbox/commit/60e7d69d6834814295c4f5d9fd40194ef814ee35 breaks on travis https://travis-ci.org/github/TIA-Lab/tiatoolbox/builds/694812115, reverting back for now until we have any other suggestion which works.

Reply:
It looks like it failed because travis doesn't have aptitude installed not because the package is wrong. Should simply be `sudo apt-get -y install openslide-tools`. `apt install` is mostly just a convenience wrapper around 'apt-get'.

Reply:
You are right, forgot to put `sudo`. Updated in this commit https://github.com/TIA-Lab/tiatoolbox/commit/2661de6172000ec304ce84268e608dafde026ed8